devops/ci/single_node_deploy/deploy.jenkinsfile

```diff
@@ -157,6 +157,11 @@ node {
                 echo "Successfully deployed system states!"
             }
             
+            stage("utils Setup") {
+                sshCommand remote: remote, command: "provisioner deploy_vm --states utils --setup-type single"
+                echo "Successfully deployed utils states!"
```

Comment:
"Successfully deployed utils states!"
to
"Successfully deployed foundation states!"